server/filter.js

```diff
@@ -55,12 +55,17 @@ function getSchemaName(event) {
 }
 
 function contains(haystack, needle) {
-    return (
-        haystack
-            .toString()
-            .toLowerCase()
-            .indexOf(needle) > -1
-    );
+    try {
+        return (
+            haystack
+                .toString()
+                .toLowerCase()
+                .indexOf(needle) > -1
+        );
+    } catch (err) {
```

Comment:
:thinking: under which circumstance may this fail? Would be good to add a comment (if avoiding it isn't possible).